code/modules/mob/living/simple_animal/hostile/gingerbreadmobs.dm

```diff
@@ -0,0 +1,392 @@
+/mob/living/simple_animal/hostile/gingerchefman
+	name = "gingerchef man"
+	desc = "Prepare to meet your baker"
+	icon_state = "gingerchefman"
+	icon_living = "gingerchefman"
+	icon_dead = "gingerchefman_dead"
+	environment_smash_flags = 0
+	wander = 0
+	turns_per_move = 15
+	move_to_delay = 3
+	speak_chance = 1
+	speak = list("Ready for the oven", "More meat for the pantry")
+	speak_emote = list("crumbles","grumbles","shouts")
+	meat_type = /obj/item/weapon/reagent_containers/food/snacks/gingerbread_man
+	response_help = "doughs up"
+	response_disarm = "crumbs"
+	response_harm = "breaks into"
+	maxHealth = 75
+	health = 75
+	attacktext = "slices"
+	melee_damage_lower = 10
+	melee_damage_upper = 30
+	attack_sound = "sound/weapons/pierce.ogg"
+	vision_range = 7
+	aggro_vision_range = 7
+	idle_vision_range = 7
+	mob_property_flags = MOB_CONSTRUCT
+	//This dough's hard baked
+	min_oxy = 0
+	max_oxy = 0
+	min_tox = 0
+	max_tox = 0
+	min_co2 = 0
+	max_co2 = 0
+	min_n2 = 0
+	max_n2 = 0
+	minbodytemp = 0
```

Comment:
This bit (and probably a couple of others) is shared between all gingerbread mobs, wouldn't it be a god idea to make them all inherit from an intermediate gingerbread mob class to keep things tidy?

Reply:
it would, yes

Reply:
As he said, no need to copypaste it.